webapp/src/components/cluster.tsx

```diff
@@ -63,6 +64,24 @@ export function Cluster() {
     );
   }
 
+  const statusRender = (text: string) => {
+      let statusColor: TagColor;
+      switch (text) {
+          case 'HEALTHY':
+              statusColor = 'green';
+              break;
+          case 'UNHEALTHY':
+              statusColor = 'red';
+              break;
+          case 'PENDING':
+              statusColor = 'yellow';
+              break;
+          default:
```

Comment:
Add case UNKNOWN before explicitly and maybe javadoc that we also default to unknown .. of check if it is even possible to get to that case then .. maybe not

Reply:
To clarify after you asked me on slack

something like (pseudo code)

```
case `UNKNOWN`:
  statusColor = 'white';
  break;
default:
  // should not happen really but in case .. defaulting to same as UNKNOWN setup
  statusColor = 'white';
  break;
```